section2/exercises/methods.rb

```diff
@@ -39,7 +56,8 @@ def print_name(name)
 
 # Look at the code you wrote for the previous YOU DO🎈 - what did you name the
 # function, and why?
-# YOUR ANSWER HERE
+puts "I put 'tough' because my back is tweeked and I'm supposed to play in a
+tournament this weekend. So I was kind of me talking to myself..."
 
 # What did you name each parameter, and why?
-# YOUR ANSWER HERE
\ No newline at end of file
+puts "I really couldn't think of what to put here. This piece tends to confuse me the most in what I should place there..."
```

Comment:
Given the method you cam up with I think those parameter names are ok. We generally want them to be more descriptive but you came up with a _tough_ example. Hope the back is ok!